engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/number/bigInteger/GreaterOrEqualNode.java

```diff
@@ -6,21 +6,20 @@
 import org.enso.interpreter.dsl.BuiltinMethod;
 import org.enso.interpreter.node.expression.builtin.number.utils.BigIntegerOps;
 import org.enso.interpreter.runtime.Context;
-import org.enso.interpreter.runtime.builtin.Builtins;
-import org.enso.interpreter.runtime.error.PanicException;
+import org.enso.interpreter.runtime.error.DataflowError;
 import org.enso.interpreter.runtime.number.EnsoBigInteger;
 
 @BuiltinMethod(type = "Big_Integer", name = ">=", description = "Comparison of numbers.")
 public abstract class GreaterOrEqualNode extends Node {
 
-  abstract boolean execute(EnsoBigInteger self, Object that);
+  abstract Object execute(EnsoBigInteger self, Object that);
 
   static GreaterOrEqualNode build() {
     return GreaterOrEqualNodeGen.create();
   }
 
   @Specialization
-  boolean doDouble(EnsoBigInteger self, double that) {
+  Object doDouble(EnsoBigInteger self, double that) {
```

Comment:
Do these `@Specialization` methods need to change return type? Can't they continue to more specialized types like `boolean` or `Atom` even the `execute` method returns `Object`?